github-merge-flow.jsonc.txt

```diff
@@ -0,0 +1,30 @@
+// IMPORTANT: This file is read by the merge flow from main branch only. 
+{
+    "merge-flow-configurations": {
+        // Automate merging runtime release/6.0 branch into release/6.0-staging
+        "release/6.0":{
+            "MergeToBranch": "release/6.0-staging",
+            "ExtraSwitches": "-QuietComments"
+        },
+        // Automate merging runtime release/8.0 branch into release/8.0-staging
+        "release/8.0":{
+            "MergeToBranch": "release/8.0-staging",
+            "ExtraSwitches": "-QuietComments"
+        },
```

Comment:
This repo does not use staging branching. Are you planning on implementing it? If not, then remove the first two entries.